static/app/views/dashboardsV2/manage/dashboardList.tsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import {browserHistory} from 'react-router';
+import styled from '@emotion/styled';
+import {Location, Query} from 'history';
+
+import WidgetArea from 'sentry-images/dashboard/widget-area.svg';
+import WidgetBar from 'sentry-images/dashboard/widget-bar.svg';
+import WidgetBigNumber from 'sentry-images/dashboard/widget-big-number.svg';
+import WidgetLine from 'sentry-images/dashboard/widget-line-1.svg';
+import WidgetTable from 'sentry-images/dashboard/widget-table.svg';
+import WidgetWorldMap from 'sentry-images/dashboard/widget-world-map.svg';
+
+import EmptyStateWarning from 'app/components/emptyStateWarning';
+import Pagination from 'app/components/pagination';
+import TimeSince from 'app/components/timeSince';
+import {t, tct} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization} from 'app/types';
+import {DashboardListItem, DisplayType} from 'app/views/dashboardsV2/types';
+
+import DashboardCard from './dashboardCard';
+
+type Props = {
+  organization: Organization;
+  location: Location;
+  dashboards: DashboardListItem[] | null;
+  pageLinks: string;
+};
+
+class DashboardList extends React.Component<Props> {
+  static miniWidget(displayType: DisplayType): string {
+    switch (displayType) {
+      case 'bar':
+        return WidgetBar;
+      case 'area':
+        return WidgetArea;
+      case 'big_number':
+        return WidgetBigNumber;
+      case 'table':
+        return WidgetTable;
+      case 'world_map':
+        return WidgetWorldMap;
+      case 'line':
+      default:
+        return WidgetLine;
+    }
+  }
+
+  renderMiniDashboards() {
+    const {organization, dashboards} = this.props;
+    return dashboards?.map((dashboard, index) => {
+      return (
+        <DashboardCard
+          key={`${index}-${dashboard.id}`}
+          title={dashboard.title}
+          to={{
+            pathname: `/organizations/${organization.slug}/dashboards/${dashboard.id}`,
+            query: {},
```

Comment:
do we need to pass query here? would it possible to pass only the pathname?

Reply:
Does this need to retain global selection state?

Reply:
Ooh, good call, yeah we probably want to retain global selection state.